src/buildercore/terraform.py

```diff
@@ -1361,58 +1418,45 @@ def init(stackname, context):
                         'credentials': "${data.%s.%s.data[\"credentials\"]}" % (DATA_TYPE_VAULT_GENERIC_SECRET, DATA_NAME_VAULT_GCP_API_KEY),
                     },
                 },
-                {
-                    'vault': {
-                        'address': context['vault']['address'],
-                        # exact version constraint
-                        'version': "= %s" % PROVIDER_VAULT_VERSION,
-                    },
-                },
-            ],
-            'data': {
-                DATA_TYPE_VAULT_GENERIC_SECRET: {
-                    # TODO: this should not be used unless Fastly is involved
-                    DATA_NAME_VAULT_FASTLY_API_KEY: {
-                        'path': VAULT_PATH_FASTLY,
-                    },
-                    # TODO: this should not be used unless GCP is involved
```

Comment:
vault is only being included if fastly is present. what gcp resources need vault? bigquery? gcs?